server/komodo-rest/src/main/java/org/komodo/rest/service/KomodoDataserviceService.java

```diff
@@ -217,7 +217,7 @@ public Response getDataservices( final @Context HttpHeaders headers,
                 throw ( KomodoRestException )e;
             }
 
-            return createErrorResponse(mediaTypes, e, DATASERVICE_SERVICE_GET_DATASERVICES_ERROR);
+            return createErrorResponse(Status.FORBIDDEN, mediaTypes, e, DATASERVICE_SERVICE_GET_DATASERVICES_ERROR);
         }
```

Comment:
So there is a place for 2 methods if you prefer:
'''
protected Response createErrorResponse(Status status, ....);
protected Response createErrorResponse(...) // assumes status forbidden
'''
The second will call the first and even prefer to rename second to something like createErrorResponseWithForbidden(...). I think it pointless to have to insert Status.FORBIDDEN is some many places to satisfy the modification of a method that could be divided into 2.